scripts/install-custom.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# -----------------------------------------------------------------------------
+# Utils
+
+build_pkgs () {
+    su --preserve-environment \
+        -c 'yes | aurbuild -d juxt -a <(aurqueue *)' rock
+}
+
+install_pkgs () {
+    # We sync pacman to pick up changes to the juxt repo.
+    pacman --noconfirm -Syu "$@"
+}
+
+# -----------------------------------------------------------------------------
+# aurutils
+
+AURUTILS_DIR="/tmp/aurutils"
+
+git clone https://aur.archlinux.org/aurutils.git "$AURUTILS_DIR"
+chmod go+w "$AURUTILS_DIR" # So we can create .SRCINFO files as the rock user.
+cd "$AURUTILS_DIR"
+
+# Install dependencies ourselves because we won't be able to elevate ourselves
+# via sudo when we `makepkg`.
+install_pkgs \
+    jq \
+    pacutils \
+    repose
+
+# You can't run makepkg as root, so we need to demote ourselves to a regular
+# user.
+#
+# We skip the PGP check because we won't have AladW's GPG key at this point.
+#
+# TODO Pull in GPG key and use it to check package.
+su --preserve-environment -c 'makepkg --skippgpcheck' rock
```

Comment:
Skipping PGP checks is me being lazy. It's the one package, but we could import `AladW`'s key and do this right perhaps?